crypto/asn1/a_strnid.c

```diff
@@ -207,10 +254,15 @@ void ASN1_STRING_TABLE_cleanup(void)
     STACK_OF(ASN1_STRING_TABLE) *tmp;
 
     tmp = stable;
-    if (tmp == NULL)
-        return;
-    stable = NULL;
-    sk_ASN1_STRING_TABLE_pop_free(tmp, st_free);
+    if (tmp != NULL) {
+        stable = NULL;
+        sk_ASN1_STRING_TABLE_pop_free(tmp, st_free);
+    }
+
+    if (stable_lock != NULL) {
```

Comment:
Don't need the NULL check here.  We'd generally not include it as a stylistic point.